packages/api/lambdas/bootstrap.js

```diff
@@ -88,7 +88,14 @@ async function bootstrapElasticSearch(host, index = 'cumulus', alias = defaultIn
     // add mapping
     await esClient.indices.create({
       index,
-      body: { mappings }
+      body: {
```

Comment:
I think it would be nice to reuse `createIndex` here rather than repeating the logic

Reply:
I started to make that change and what i didn't love about it is you do the exists check right above so you'd be doing it twice.  And then you could do `createIndex` and then handle the error and set `exists` that way, but that made it seem like you shoehorned `createIndex` in. And then it's like well you could pull the actual creation piece out to its own function which seemed like overengineering. I hope you enjoyed this recap of the conversation I had with myself.

Reply:
I guess I don't think pulling the code just doing the index creation into its own function is overengineering and it avoids future drift between the code using this new env vars. The current `createIndex` helper is overloaded in that it checks for the index and creates the index. If those were two separate functions, then all of the calling code could implement them as desired (throw exceptions or not, etc). 

I don't feel super strongly, but I think it would be nice. But if you think this is out of scope we can nix it for now.